infrastructure/variables.tf

```diff
@@ -146,6 +146,10 @@ variable "elastic_search_enabled" {
   default = "false"
 }
 
+variable "elastic_search_request_timeout" {
```

Comment:
you have to modify the `main.tf` to assign this var value to an environment variable

Reply:
DONE